src/components/sporsmal/tilleggssporsmal/Egenmeldingsdager.tsx

```diff
@@ -96,49 +82,49 @@ const Egenmeldingsdager = ({
                 <Fieldset legend={tekster['egenmeldingsperioder.tittel']}>
                     {perioder.map(periode => {
                         return (
-                            <div key={periode.id}>
-                                <Periodevelger
-                                    vis={true}
-                                    id={periode.id}
-                                    minDato={new Date('12.01.2019')} // TODO: lage logikk for å intervallbegrensning
-                                    maksDato={new Date('12.10.2019')}
-                                    setValue={updateValue}
+                            <div className="periode" key={periode.id}>
+                                <Flatpickr
+                                    value={periode.datoer}
+                                    className="typo-normal flatpickr"
+                                    placeholder="Trykk for å velge periode"
+                                    onChange={datoer => oppdaterPeriode(periode.id, datoer)}
+                                    options={{
+                                        minDate: new Date('10.02.2019'),
+                                        maxDate: new Date('11.10.2019'),
+                                        mode: 'range',
+                                        enableTime: false,
+                                        dateFormat: 'd-m-y',
+                                        altInput: true,
+                                        altFormat: 'j. M, Y',
+                                        locale: locale,
+                                    }}
                                 />
+                                {/* Skal ikke kunne slette første periode */}
                                 {periode.id !== 0 && (
-                                    <Knapp
-                                        type={'fare'}
-                                        form={'kompakt'}
-                                        mini
+                                    <Lenke
```

Comment:
Skikkelig småpirk, men jeg misliker at lenker brukes som knapper (altså at de ikke navigerer deg videre noe sted). Man får opp en lenkepreview nederst til venstre i chrome. Mulig at skjermlesere også identifiserer dette som en lenke i stedet for en knapp.

Reply:
Godt poeng. Syns å huske at det var gjort slik i den forrige. Tror vi skal høre med bendik og kjell arne hva de tenker. Kan eventuelt designe en ny komp hvis den må se lik ut som forrige design

Reply:
Greit! Lag gjerne en issue på det så vi husker det.